build: Bump to version 0.14 (#7)
jungsoft/paper_trail: CHANGELOG.md

> ### v0.14 - August 24th, 2022 - Jungsoft version:

Reviewer: Wrong date here

> - Avoid empty association updates

Reviewer: ```suggestion
- Do not serialize nested associations when action is `update` and changes are empty
```

> - Add event and data infos when replace association

Reviewer: ```suggestion
- Add `event` and `data` info when serializing an association with `action == :replace`
```

> - Add event and changes infos in all other association events

Reviewer: ```suggestion
- Add `event` and `changes` info when serializing an association with `action != :replace`
```
